front-end/src/components/Profile/TeacherInfo/TeacherInfo.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useContext, useState, useEffect } from 'react';
+
+// context
+import { CurrentUserContext } from 'context/AppProvider';
+
+// services
+import { getTeacher, updateTeacher } from '../../../services/teacherService';
+
+// styles
+import styles from './TeacherInfo.module.scss';
+import SelectItem from './SelectItem/SelectItem';
+import TextAreaInfo from './TextAreaItem/TextAreaInfo';
+import TeacherPhone from './TeacherPhone/TeacherPhone';
+import { Button } from '@mui/material';
+
+const languages = ['English', 'German', 'Italian'];
+const operators = ['+38', '+98'];
+const agePreferences = ['6 - 10', '11 - 15', '16 - 17', '18 - ...'];
+
+const TeacherInfo = () => {
+  const { currentUser } = useContext(CurrentUserContext);
+  console.log('user', currentUser);
+  const [language, setLanguage] = useState('');
+  const [biography, setBiography] = useState('');
+  const [phone, setPhone] = useState(0);
```

Comment:
Inside teacher model field phone have type of string, maybe instead of 0 you should have ''.